Documentation/gettingstarted/rancher.rst

```diff
@@ -0,0 +1,95 @@
+****************************
+Installing Cilium on Rancher
+****************************
+
+Create a New Cluster
+=====================
+
+In Rancher UI, navigate to the Clusters page. In the top right, click on the
+`Add Cluster` box to create a new cluster.
+
+.. image:: images/rancher_add_cluster.png
+
+On the Add Cluster page select to create a new cluster from `Existing Nodes`:
+
+.. image:: images/rancher_existing_nodes.png
+
+On the `Add Cluster` page that opens, provide a name for the cluster. Next to
+the `Cluster Options` section click the box to `Edit as YAML`. The configuration
+for the cluster will open up in an editor in the window.
+
+.. image:: images/rancher_edit_as_yaml.png
+
+Remove all configuration pertaining to the default network plugin:
+
+.. image:: images/rancher_delete_network_plugin.png
+
+Ensure that network plugin type `none` is specified:
+
+.. image:: images/rancher_network_plugin_none.png
+
+Make any additional changes to the configuration that are appropriate for your
+environment. When you are ready, click `Create` and Rancher will create the
+cluster in the `Provisioning` state.
+
+.. image:: images/rancher_cluster_state_provisioning.png
+
+Rancher will also present you with instructions on how to add nodes to the
+cluster:
+
+.. image:: images/rancher_add_nodes.png
+
+Before adding nodes to the cluster, ensure the BPF filesystem  is mounted.
+You can persist the configuration using the following commands:
+
+```
+sudo mount bpffs -t bpf /sys/fs/bpf
+sudo bash -c 'cat <<EOF >> /etc/fstab
+none /sys/fs/bpf bpf rw,relatime 0 0
+EOF'
+```
```

Comment:
This is Markdown. RST:
```suggestion
.. code-block:: shell-session

    $ sudo mount bpffs -t bpf /sys/fs/bpf
    $ sudo bash -c 'cat <<EOF >> /etc/fstab
    none /sys/fs/bpf bpf rw,relatime 0 0
    EOF'
```

Can't the second command be simplified into something like `sudo echo 'none /sys/fs/bpf bpf rw,relatime 0 0' >> /etc/fstab`?